pkg/scheduler/ugm/queue_tracker.go

```diff
@@ -113,6 +113,7 @@ func (qt *QueueTracker) increaseTrackedResource(hierarchy []string, applicationI
 		qt.resourceUsage = resources.NewResource()
 	}
 	qt.resourceUsage.AddTo(usage)
+	qt.resourceUsage.Prune()
```

Comment:
The alloc should not have zeros in it. The only prune that is needed is when we do the `Sub` or `SubFrom`

Reply:
Addressed in latest PR, thanks!